test/functional/bip68-sequence.py

```diff
@@ -369,11 +369,14 @@ def test_bip68_not_consensus(self):
 
     def activateCSV(self):
         # activation should happen at block height 432 (3 periods)
+        # getblockchaininfo will show CSV as active at block 431 (144 * 3 -1) since it's returning whether CSV is active for the next block.
         min_activation_height = 432
         height = self.nodes[0].getblockcount()
-        assert(height < min_activation_height)
-        self.nodes[0].generate(min_activation_height-height)
-        assert(get_bip9_status(self.nodes[0], 'csv')['status'] == 'active')
+        assert_greater_than(min_activation_height - height, 2)
```

Comment:
Definitely a matter of personal taste, but:

```python
assert_greater_than(min_activation_height - 2, height)
```

seems to me to convey the meaning better: that the current height is less than (activation height - 2).